nlp_primitives/tests/test_median_word_length.py

```diff
@@ -0,0 +1,42 @@
+import numpy as np
+import pandas as pd
+
+from ..median_word_length import MedianWordLength
+from .test_utils import PrimitiveT, find_applicable_primitives, valid_dfs
+
+
+class TestMedianWordLength(PrimitiveT):
+    primitive = MedianWordLength
+
+    def test_delimiter_override(self):
+        x = pd.Series(['This is a test file.',
+                       'This,is,second,line?',
+                       'and;subsequent;lines...'])
+
+        expected = pd.Series([4., 4.5, 8.])
+        actual = self.primitive('[ ,;]').get_function()(x)
+        pd.testing.assert_series_equal(actual, expected, check_names=False)
+
+    def test_multiline(self):
+        x = pd.Series(['This is a test file.',
+                       'This is second line\nthird line $1000;\nand subsequent lines'])
+
+        expected = pd.Series([3.0, 4.0])
```

Comment:
I think we are getting some empty strings during the splitting when we have consecutive delimiters that are impacting the results. I think the median word length for `This is a test file.` should be 4, not 3, no? There are 5 words with lengths `1, 2, 4, 4, 4`.

Maybe need to update `[len(word) for word in words]` to `[len(word) for word in words if len(word) != 0]` in the `get_median` function.

Reply:
yeah, makes sense --updated